src/box/xlog.c

```diff
@@ -731,7 +731,7 @@ xdir_collect_inprogress(struct xdir *xdir)
 		if (ext == NULL || strcmp(ext, inprogress_suffix) != 0)
 			continue;
 
-		char path[PATH_MAX];
+		char path[PATH_MAX * 2];
```

Comment:
Why?

Reply:
Discussed f2f: the compiler warning should be ignored.